skydriver/database/__init__.py

```diff
@@ -1,3 +1,23 @@
 """Init."""
 
-from . import interface, schema  # noqa: F401
+
+from urllib.parse import quote_plus
+
+from motor.motor_asyncio import AsyncIOMotorClient  # type: ignore
+
+from ..config import ENV
+from . import interface
+
+
+async def create_mongodb_client() -> AsyncIOMotorClient:
+    """Construct the MongoDB client."""
+    auth_user = quote_plus(ENV.MONGODB_AUTH_USER)
+    auth_pass = quote_plus(ENV.MONGODB_AUTH_PASS)
+
+    url = f"mongodb://{ENV.MONGODB_HOST}:{ENV.MONGODB_PORT}"
+    if auth_user and auth_pass:
+        url = f"mongodb://{auth_user}:{auth_pass}@{ENV.MONGODB_HOST}:{ENV.MONGODB_PORT}"
```

Comment:
Should this be an if/else, instead of redefining the url?

Reply:
sure, [0edb1e4](https://github.com/WIPACrepo/SkyDriver/pull/36/commits/0edb1e4f050b10c04b5fbb71d260600aa148fb48)